pkg/gator/test/test_test.go

```diff
@@ -168,27 +182,20 @@ func TestTest(t *testing.T) {
 			var objs []*unstructured.Unstructured
 			for _, input := range tc.inputs {
 				u, err := readUnstructured([]byte(input))
-				if err != nil {
-					t.Fatalf("readUnstructured for input %q: %v", input, err)
-				}
+				require.NoError(t, err)
 				objs = append(objs, u)
 			}
 
 			resps, err := Test(objs, false)
 			if tc.err != nil {
-				if err == nil {
-					t.Errorf("got nil err, want %v", tc.err)
-				}
-				if !errors.Is(err, tc.err) {
-					t.Errorf("got err %q, want %q", err, tc.err)
-				}
+				require.ErrorIs(t, err, tc.err)
 			} else if err != nil {
-				t.Errorf("got err '%v', want nil", err)
+				require.NoError(t, err)
 			}
 
 			got := resps.Results()
 
-			diff := cmp.Diff(tc.want, got, cmpopts.IgnoreFields(GatorResult{}, "Metadata", "EvaluationMeta", "EnforcementAction", "ViolatingObject"))
+			diff := cmp.Diff(tc.want, got, ignoreGatorResultFields())
```

Comment:
Did you want to update https://github.com/open-policy-agent/gatekeeper/blob/9ddb3a6c515fce39c4752b543f81f187bbc21928/pkg/gator/test/test_test.go#L254-L259 as well?

Reply:
@acpana _bump_

Reply:
He'll be back a little later this month

Reply:
@ritazh thanks for highlighting this & your patience; I changed it now 🚀